selfdrive/boardd/boardd.cc

```diff
@@ -392,8 +376,47 @@ void panda_state_thread(Panda *&panda, bool spoofing_started) {
   }
 }
 
-void hardware_control_thread(Panda *panda) {
-  LOGD("start hardware control thread");
+void peripheral_state_thread(Panda *panda) {
```

Comment:
this should stay in the panda_state_thread. we don't really want the scheduler to handle this for us